module/VuFind/src/VuFind/Db/Table/Session.php

```diff
@@ -137,9 +137,7 @@ public function writeSession($sid, $data)
     public function destroySession($sid)
     {
         $s = $this->getBySessionId($sid, false);
-        if (!empty($s)) {
-            $s->delete();
-        }
+        $s->delete();
```

Comment:
getBySessionId is not guaranteed to return an object; I think we need to fix the return type on that function rather than change the logic here.

Reply:
You are right, I changed the return type of getSessionById